doc/api/perf_hooks.md

```diff
@@ -165,6 +165,30 @@ obs.observe({ entryTypes: ['function'] });
 wrapped();
 ```
 
+### `performance.eventLoopUtilization([util])`
+<!-- YAML
+added:
+-->
+
+* `util` {Object} The result of a previous call to `eventLoopUtilization()`
+* Returns {Object}
+  * `idle` {number}
+  * `active` {number}
+  * `utilization` {number}
+
+The `eventLoopUtilization()` method returns an object that contains the
+cumulative duration of time the event loop has been both idle and active as a
+high resolution milliseconds timer. The event loop utilization as a percentage
+is calculated in the `utilization` value.
+
```

Comment:
a one liner on loop idleness would be useful here: I guess this is the wait period in `poll`?